lib/validator.js

```diff
@@ -82,6 +82,9 @@ export default class Validator {
 
         const run = (v, rules, triedToConvert = false) => {
             const rule = rules[0];
+            if (v === undefined && rule._default) {
+                v = rule._default;
+            }
```

Comment:
I would define a more general "options" rule that has the ability to change how the validation is run before we start looping over rules. This pattern seems like it could become spaghetti :stuck_out_tongue: